pages/base_page.py

```diff
@@ -1,4 +1,5 @@
 from selenium import webdriver
+from selenium.common import TimeoutException
```

Comment:
I'd like to refer here to this part od code:
```python
    class __WebDriver:
        def __init__(self):
            self.driver = webdriver.Chrome(executable_path=ChromeDriverManager().install())
    driver = None
    def __init__(self):
        if not self.driver:
            BasePage.driver = BasePage.__WebDriver().driver
        self.explicitly_wait = WebDriverWait(
            driver=self.driver,
            timeout=5
        )
```
Basically it should work but this should work the same:
```python
    driver = None

    def __init__(self):
        if not self.driver:
            BasePage.driver = webdriver.Chrome(executable_path=ChromeDriverManager().install())
        self.explicitly_wait = WebDriverWait(
            driver=self.driver,
            timeout=5
        )
```
Internal class in completely redundant and increases code complexity